src/components/InfinityModal/InfinityModalForm.tsx

```diff
@@ -0,0 +1,330 @@
+						component={renderInput}
+					/>
+
+				</div>
+				<div className="infinity-field">
+					<Field
+						id="tn"
+						name="tn"
+						label={tnLabel}
+						component={renderInput}
+					/>
+
+				</div>
+			</div>
+			<div className="skill-level-field infinity-skill-level-field">
+				<Field
+					id="difficulty"
+					name="difficulty"
+					component={DiffLadder}
+				/>
+			</div>
+			<div className="flex-center">
+				<div className="infinity-radio-fields infinity-radio-fields--dice-to-roll">
+					<InfinityDiceRow
+						dice={dice}
+						diceMax={5}
+						fortune={fortune}
+						hoverState={hoverState}
+						handleOnHover={setHoverState}
+						handleOnClick={handleDiceChange}
+						isAssistance={false}
+					/>
+				</div>
+			</div>
+			<Field
+				name="untrainedTest"
+				id="untrainedTest"
+				label={untrainedTestLabel}
+				component={RenderCheckbox}
+				disabled={focus && Number(focus) > 0}
+			/>
+			<div className="fortune">
+				<h5 className="fortune-title">Fortune <InfoTooltip content={tooltip.fortuneInfo} /></h5>
+				<div className="infinity-radio-fields">
+					<label className="dice-row-label">
+						<Field
+							name="fortune"
+							component="input"
+							type="radio"
+							value="0"
+						/>
+							None
+						</label>
+					<label className="dice-row-label">
+						<Field
+							name="fortune"
+							component="input"
+							type="radio"
+							value="1"
+							onChange={() => Number(dice) < 3 ? change('dice', '3') : null}
+						/>
+							One
+						</label>
+					<label className="dice-row-label">
+						<Field
+							name="fortune"
+							component="input"
+							type="radio"
+							value="2"
+							onChange={() => Number(dice) < 4 ? change('dice', '4') : null}
+						/>
+							Two
+						</label>
+					<label className="dice-row-label">
+						<Field
+							name="fortune"
+							component="input"
+							type="radio"
+							value="3"
+							onChange={() => Number(dice) < 5 ? change('dice', '5') : null}
+						/>
+							Three
+						</label>
+				</div>
+			</div>
+			<div className="assistance">
+				<Accordion defaultActiveKey="0">
+					<Accordion.Toggle className="assistance-title--container" eventKey="1">
+						<h5 className="assistance-title">Assistance <InfoTooltip content={tooltip.assistanceInfo} /></h5>
+					</Accordion.Toggle>
+					<Accordion.Collapse eventKey="1">
+						<>
+							<div className="flex-center">
+								<div className="infinity-radio-fields infinity-radio-fields--dice-to-roll infinity-radio-fields--assistance">
+									<InfinityDiceRow
+										dice={assistanceDice}
+										diceMax={4}
+										fortune={0}
+										hoverState={assistanceHover}
+										handleOnHover={setAssistanceHover}
+										handleOnClick={handleAssistanceDiceChange}
+										isAssistance={true}
+									/>
+								</div>
+							</div>
+							<div className="skill-level-field infinity-skill-level-field infinity-skill-level-field--assistance">
```

Comment:
Number of these classes scares me. Try to compress them into one or two, with css modules it should be easier.

Reply:
true, I did this as Conan was set up due to time constraints. it is up to you if you need them as modules before merge or can accept as is. I am not familiar with css.modules so I may not get to that change for a while.